Project-Hotpot/Project-Hotpot/Parse-Backend/PFPlaylist.swift

```diff
@@ -28,17 +28,20 @@ class PFPlaylist: PFObject, PFSubclassing {
         newPlaylist.name = name
         newPlaylist.songArray = []
         
-        do {
-            //save synchronously, since creation will most likely be followed by edits to the playlist
-            //TODO: Feedback on synchronous creation or async? If synchronous, is completion necessary?
-            try newPlaylist.save()
-        }
-        catch{
-            NSLog("Error saving new playlist: \(error)")
-            completion(false)
-        }
-        completion(true)
+        //save asynchronously
+        newPlaylist.saveInBackground(block: {isSuccessful, error in
+            if (isSuccessful){
+                completion(true)
+            }
+            else{
```

Comment:
Another small bump to get into the habit of adding proper spacing around your brackets.